subprojects/docs/src/docs/userguide/distributionPlugin.adoc

```diff
@@ -18,37 +18,37 @@
- 
+
+The Distribution Plugin is currently <<feature_lifecycle,incubating>>. Please be aware that the DSL and other configuration may change in later Gradle versions.
+
 ====
 
-The distribution plugin facilitates building archives that serve as distributions of the project. Distribution archives typically contain the executable application and other supporting files, such as documentation.
+The Distribution Plugin facilitates building archives that serve as distributions of the project. Distribution archives typically contain the executable application and other supporting files, such as documentation.
 
 
 [[sec:distribution_usage]]
 === Usage
 
-To use the distribution plugin, include the following in your build script:
+To use the Distribution Plugin, include the following in your build script:
 
 ++++
-<sample id="useDistributionPlugin" dir="userguide/distribution" title="Using the distribution plugin">
-            <sourcefile file="build.gradle" snippet="use-plugin"/>
-        </sample>
+<sample id="useDistributionPlugin" dir="userguide/distribution" title="Using the Distribution Plugin">
+    <sourcefile file="build.gradle" snippet="use-plugin"/>
+</sample>
 ++++
 
-The plugin adds an extension named “`distributions`” of type api:org.gradle.api.distribution.DistributionContainer[] to the project. It also creates a single distribution in the distributions container extension named “`main`”. If your build only produces one distribution you only need to configure this distribution (or use the defaults).
+The plugin adds an extension named `distributions` of type api:org.gradle.api.distribution.DistributionContainer[] to the project. It also creates a single distribution in the distributions container extension named `main`. If your build only produces one distribution you only need to configure this distribution (or use the defaults).
 
-You can run “`gradle distZip`” to package the main distribution as a ZIP, or “`gradle distTar`” to create a TAR file. To build both types of archives just run `gradle assembleDist`. The files will be created at “`__$buildDir__/distributions/__$project.name__-__$project.version__.__«ext»__`”.
+You can run `gradle distZip` to package the main distribution as a ZIP, or `gradle distTar` to create a TAR file. To build both types of archives just run `gradle assembleDist`. The files will be created at `__$buildDir__/distributions/__$project.name__-__$project.version__.__«ext»__`.
 
-You can run “`gradle installDist`” to assemble the uncompressed distribution into “`__$buildDir__/install/__$project.name__`”.
+You can run `gradle installDist` to assemble the uncompressed distribution into `__$buildDir__/install/__$project.name__`.
 
 [[sec:distribution_tasks]]
 === Tasks
 
-The Distribution plugin adds the following tasks to the project:
+The Distribution Plugin adds the following tasks to the project:
 
-.Distribution plugin - tasks
+.Distribution Plugin - tasks
```

Comment:
❓ Huge optional favor: could you please reformat this table to something more like https://docs.gradle.org/current/userguide/java_plugin.html#sec:java_tasks?

This fixes the display for medium-sized screens and IMO is prettier and more usable.

Reply:
✅ Done.